File: Service_eHospital_DongNai_A/Implement/CDT_LichCongTacService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CDT_LichCongTacService : BaseService<CDT_LichCongTac, ICDT_LichCongTacRepository>
    , ICDT_LichCongTacService
    {
    private readonly ICDT_LichCongTacRepository _CDT_LichCongTacRepository;
    public CDT_LichCongTacService(ICDT_LichCongTacRepository CDT_LichCongTacRepository) : base(CDT_LichCongTacRepository)
    {
    _CDT_LichCongTacRepository = CDT_LichCongTacRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/CDT_LichCongTac.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CDT_LichCongTac } from './CDT_LichCongTac.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CDT_LichCongTacService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CDT_LichCongTac_Id', 'CDT_Loai_Id', 'NoiDungCongViec', 'NgayThucHien', 'ThoiGianThucHien', 'ThanhPhanThamGiaDuKien', 'KinhPhiDuKien', 'NguoiLap_Id', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: CDT_LichCongTac[] | undefined;
    ListFilter: CDT_LichCongTac[] | undefined;
    FormData!: CDT_LichCongTac;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CDT_LichCongTac";
    }
}

